templates/index.html

```
@@ -26,467 +31,27 @@ <h1 class="logo">
-                      <img src="./images/star_empty.png" class="star" /><span
-                        >7.7</span
-                      >
-                    </p>
-                    <strong>인사이드 아웃 2</strong>
-                  </div>
-                </div>
-              </li>
-            </ul>
+            <p class="page-title">지금 인기 있는 영화</p>
+            <div class="search-error-container" hidden>
+              <img src="" alt="" />
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)